firefox-ios/firefox-ios-tests/Tests/ClientTests/Mocks/MockDataCleaner.swift

```diff
@@ -0,0 +1,27 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Foundation
+import Shared
+@testable import Client
+
+class MockCookiesClearable: Clearable {
+    var label: String { .ClearableCookies }
+    var isSucceed: Success?
+
+    func clear() -> Success {
+        isSucceed = succeed()
+        return succeed()
+    }
+}
+
+class MockSiteDataClearable: Clearable {
```

Comment:
Same here, curious if we make this `SiteDataClearable`?

Reply:
I am thinking of expanding `class SiteDataClearable: Clearable {` so it has some inject able for the `func clear`

Reply:
Makes sense also for using SiteDataClearable instead of Clearable but for the second suggestion, I am not really sure about it or how it may look.